<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Doctor_inf</class>
 <widget class="QDialog" name="Doctor_inf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-90</x>
     <y>388</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>241</width>
     <height>283</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Імя</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Label">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Прізвище</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="LineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Label_2">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Посада</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="LineEdit_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="LineEdit_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="Label_4">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Телефон</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="LineEdit_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="Label_5">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Години прийому</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="LineEdit_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="Label_6">
      <property name="text">
       <string>Рейтинг</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="LineEdit_6">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="Label_7">
      <property name="text">
       <string>Кількість оцінок</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="LineEdit_7">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="Label_3">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Кабінет</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     border-image: url(:/search/search.png);
	font: 75 oblique 11pt &quot;Norasi&quot;;
	outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>33</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     border-image: url(:/search/search.png);
	font: 75 oblique 11pt &quot;Norasi&quot;;
	outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>99</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     border-image: url(:/search/search.png);
	font: 75 oblique 11pt &quot;Norasi&quot;;
	outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>275</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>356</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     
	background-color: rgb(245, 245, 245);

	font: 75 oblique 11pt &quot;Norasi&quot;;
	outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>356</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     
	background-color: rgb(245, 245, 245);
	font: 75 oblique 11pt &quot;Norasi&quot;;
	outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>356</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     
	background-color: rgb(245, 245, 245);
	font: 75 oblique 11pt &quot;Norasi&quot;;
	outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>356</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     
	background-color: rgb(245, 245, 245);
	font: 75 oblique 11pt &quot;Norasi&quot;;
	outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>356</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     
	background-color: rgb(245, 245, 245);

	font: 75 oblique 11pt &quot;Norasi&quot;;
	outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>309</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>309</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>309</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>197</x>
     <y>309</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>308</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>345</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Doctor_inf</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Doctor_inf</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
